<launch>
<!-- Launch file for Livox AVIA LiDAR -->

	<arg name="rviz" default="false" />

	<rosparam command="load" file="$(find fast_lio)/config/avia.yaml" />

	<param name="pcd_path" type="string" value="$(find robots_localization)/PCD/2024-05-20_19-48-33_scans.pcd" />
    <!-- add param for manual select map pcd file -->
    <node pkg="robots_localization" type="robots_localization" name="robots_localization" output="screen" /> 

    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz"
            args="-d $(find robots_localization)/rviz/localization.rviz" />
    </group>

</launch>
